VideoWeb/VideoWeb/Controllers/ConferencesController.cs

```diff
@@ -168,6 +176,21 @@ public async Task<ActionResult<List<ConferenceForVhOfficerResponse>>> GetConfere
             }
         }
 
+        private void UpdateConferencesWithAllocatedCsos(ICollection<AllocatedCsoResponse> allocatedHearings, List<ConferenceForVhOfficerResponse> responses)
+        {
+            foreach (var hearings in allocatedHearings)
+            {
+                var conference = responses.FirstOrDefault(conference => hearings.HearingId == conference.HearingRefId);
+                if (conference == null)
+                {
+                    _logger.LogWarning("Allocated hearing id, not in list of conferences for response");
+                    continue;
+                }
+
+                conference.AllocatedCso = hearings?.Cso?.FullName ?? "Unallocated";
```

Comment:
I think here we should have a logic for the labelling. There is a logic for displaying the right label in the hearing mapper.